project_one/index.html

```diff
@@ -0,0 +1,82 @@
+
+<html>
+<head>
+ <title> Soul BarberShop  </title> 
+ <link href="style.css" rel="stylesheet" type="text/css">
+
+</head>
+
+<body>
+  <div >
+
+      <!-- <button id ="grading-element-one"  onclick="myFunction()">yes</button> -->
+
+        <div>
+            <!-- <script type ="text/javascript">
+             
+               let hairStyle =[{hairStyle1 : 'Low Fade'} ,{hairStyle1:'High fade'},{hairStyle1: 'Mid fade'},{hairStyle1: 'high Top Fade'},{hairStyle1:'Crew Cut'}];
+                
+                 const getNewArray =(item) =>{
+            
+                    const updateArray ="<ol>"+ [item.hairStyle1, item.hairStyle]+"</ol>";
+                
+                    return updateArray;
+            
+                        
+                    console.log(updateArray);
+            
+                }
+            
+                const barberNameHandler= () =>{
+            
+                    document.getElementById("Showlist").innerHTML = hairStyle.map(getNewArray);}
+            </script> -->
+
+            
+                
+              </ul>
+              
+              <p>Click the button if you would like to give a donnation</p>
+              
+              <button id ="grading-element-one"  onclick="myFunction()">yes </button>
+
+              <ul id="myList"></ul>
+              
+              <script  id="grading-element-one-consequence">
```

Comment:
though this can work, generally you want to put your `script` tags at the bottom of the `body` element, as the last of the children of `body`.

Reply:
professor i have been trying to push my last changes and make a pull reques for the past 45 mns unfortunatly iwasnt able do to so 
<img width="847" alt="Screen Shot 2019-10-23 at 12 18 13 AM" src="https://user-images.githubusercontent.com/43247361/67357326-f20ed280-f52a-11e9-9414-6a0a463509fb.png">

Reply:
i'm ready to grade this but it looks like you have not updated since...?

Reply:
I'M MAKING A NEW PULL REQUEST NOW FOR THE EXTRA CREDIT I USED FLEXBOX CSS AND THRE MORE PAGES